src/query.js

```diff
@@ -147,7 +148,7 @@ function repo(req) {
     return "";
   }
 
-  const re = /^[a-zA-Z0-9\-][\w\-.]{0,213}\/[a-zA-Z0-9\-][\w\-.]{0,213}$/;
+  const re = /^[a-zA-Z\d-][\w-.]{0,213}\/[a-zA-Z\d-][\w-.]{0,213}$/;
```

Comment:
Unfortunately the changes here, result in invalid regex. `-` standalone attempts to create a range, meanwhile `\-` indicates `-` literally character, which is what we want to do. Otherwise if we move `-` to the last position of the character class, it becomes a standard character to match within the set. So either we should add back in `\-` or create a result that looks like this `^[a-zA-Z\d-][\w.-]{0,213}\/[a-zA-Z\d-][\w.-]{0,213}$`. Your choice as to which method looks best.

Reply:
And looking through your commit history I see this was removed under `unnecessary escape character for dash` and just wanted to let you know I'm basing my comment on the results from [regex101.com](https://regex101.com) that produces this with those changes: 
![image](https://user-images.githubusercontent.com/26921489/179637642-c5a0ced6-d6a4-43ec-8816-fa2462468e40.png)

Reply:
My bad, I ended up removing the escape also where I didn't had to.

Anyway, now that I think about it, having a dash not escaped at the end of the group could be misleading if in the future we want to add a new character (which generally is put at the end, it may create an unwanted range). So it's better to have it at the start.

Reply:
I agree with you there, thats a good thought to save ourselves an accident bug down the road. And I see you just fixed it with this new suggestion. Beautiful, I'll go ahead and merge to main